pkg/devspace/build/builder/helper/util.go

```diff
@@ -111,7 +116,11 @@ func RewriteDockerfile(dockerfile string, entrypoint []string, cmd []string, tar
 
 		if len(entrypoint) == 0 {
 			if len(oldEntrypoint) == 0 {
-				return "", errors.Errorf("cannot inject restart helper into dockerfile because no ENTRYPOINT was found, please make sure your dockerfile has an ENTRYPOINT defined or define it in the devspace.yaml in 'images.*.entrypoint'")
+				if len(oldCmd) == 0 {
+					return "", errors.Errorf("Cannot inject restart helper into Dockerfile because neither ENTRYPOINT nor CMD was found.\n\nHow to fix this:\n- Option A: Define an ENTRYPOINT (or CMD) in your Dockerfile\n- Option B: Set `images.*.entrypoint` option in your devspace.yaml")
```

Comment:
We should use lowecase error strings as this is a golang best practice: https://github.com/golang/go/wiki/CodeReviewComments#error-strings